### DISCUSSION TITLE
Blocco aree registro

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#443] [Idee e proposte]
Salve, sarebbe possibile, per l'infanzia e elementari bloccare per le famiglie la sezione voti?

**Comment by trinko** [comment]
Al momento non c'è una gestione che permetta di farlo.
Si può fare una modifica al template che gestisce i menu, eliminado la voce dei voti. Ma tieni presente che la pagina sarebbe comunque visibile se qualcuno viene a conoscenza della url corrispondente alla visualizzazione dei voti e la inserisce direttamente...
